webpackBaseConfig.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
       '@mui/material-next': path.resolve(__dirname, './packages/mui-material-next/src'),
       '@mui/material-nextjs': path.resolve(__dirname, './packages/mui-material-nextjs/src'),
       '@mui/joy': path.resolve(__dirname, './packages/mui-joy/src'),
-      '@mui/zero-runtime': path.resolve(__dirname, './packages/zero-runtime/src'),
+      '@pigmentcss/react': path.resolve(__dirname, './packages/pigment-react/src'),
```

Comment:
Could we exactly match the npm package name, to avoid confusion, the length of it doesn't seem to matter. It's also one less thing we need to decide on.

```suggestion
      '@pigmentcss/react': path.resolve(__dirname, './packages/pigmentcss-react/src'),
```